app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,46 @@
+class MoviesController < ApplicationController
+  def index
+    movies = Movie.all
+
+    render(
+      json: movies.as_json(only: [:id, :title, :release_date]),
+      status: :ok
+    )
+  end
+
+  def show
+    movie = Movie.find_by(id: params[:id])
+    if movie
+      render(
+        json: movie.as_json(only:[ :title, :overview, :release_date, :inventory, :available_inventory]),
+        status: :ok
+      )
+    else
+      render(
+        json:{nothing: "true"},
+        status: :not_found
+      )
```

Comment:
This will literally send back `{ nothing: true }` as a JSON response. Not sure that's what you intended.